test/unit/org/apache/cassandra/db/ColumnFamilyStoreTest.java

```diff
@@ -1394,6 +1408,469 @@ public void testRangeSlicePaging() throws Throwable
         assertColumnNames(row2, "c1");
     }
 
+    @Test
+    public void testRangeSlicePageTokenVariousSlices()
+    {
+        String keyspaceName = KEYSPACE1;
```

Comment:
is there any way you could make these tests more readable? Perhaps more, shorter test cases, and some util methods for constructing the initial state.

Reply:
You can also experiment with inner classes for test suites, so that a set of tests can all share the same @BeforeEach method. Or just put the tests for page token range scans in a separate test class.

Reply:
Specifically, I think each assertion where you've commented a brief description of the test case should really be a separate `@Test` method.